package.json

```diff
@@ -192,5 +192,6 @@
         "@semantic-release/github"
       ]
     ]
-  }
+  },
+  "packageManager": "yarn@3.6.1+sha256.811210abb5fb5751da12ead8a9cbc0c150b07e43ac9cbedec6752d22abfd2bd6"
```

Comment:
necessary?

Reply:
My bad, newer yarn cli tends to add this automatically. Removed
https://github.com/Qovery/console/pull/1433/commits/ac99ffdd49c46ffaa5d58b45f33315f67a19329d